msal4j-sdk/pom.xml

```diff
@@ -0,0 +1,276 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>com.microsoft.aad.msal4j</artifactId>
+        <groupId>com.microsoft.azure</groupId>
+        <version>1.13.1</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>com.microsoft.aad.msal4j-sdk</artifactId>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
I don't think we were specifying this in our current pom.xml, but this should probably be 1.8 or 8 instead of 11

Reply:
sounds good. I removed it altogether.